code/modules/clothing/outfits/ert.dm

```diff
@@ -154,30 +151,29 @@
 /datum/outfit/centcom/ert/engineer/alert
 	name = "ERT Engineer - High Alert"
 
-	mask = /obj/item/clothing/mask/gas/sechailer/swat
-	backpack_contents = list(/obj/item/storage/box/survival/engineer=1,\
-		/obj/item/melee/baton/loaded=1,\
-		/obj/item/gun/energy/pulse/pistol/loyalpin=1,\
-		/obj/item/construction/rcd/combat=1,\
+	backpack_contents = list(/obj/item/storage/box/survival/engineer=1,
+		/obj/item/melee/baton/loaded=1,
+		/obj/item/gun/energy/pulse/pistol/loyalpin=1,
+		/obj/item/construction/rcd/combat=1,
 		/obj/item/pipe_dispenser=1)
 
-
 /datum/outfit/centcom/centcom_official
 	name = "CentCom Official"
 
+	id = /obj/item/card/id/advanced/centcom
+	id_trim = /datum/id_trim/centcom/official
 	uniform = /obj/item/clothing/under/rank/centcom/officer
-	shoes = /obj/item/clothing/shoes/sneakers/black
-	gloves = /obj/item/clothing/gloves/color/black
+	back = /obj/item/storage/backpack/satchel
+	backpack_contents = list(/obj/item/storage/box/survival/engineer=1,
```

Comment:
?

Reply:
Sorry I took longer to check on this, but I'm not sure what you mean with that.
If it's the added engineer survival box it's mostly because I assume it was an oversight that they didn't have any as even Interns have it the basic one and I will change to that then.